assets/app/scripts/extensions/javalink/javaLink.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+(function() {
```

Comment:
@gashcrumb agree, like the IIFE wrapper to avoid globals.  Didn't pass in `angular` or `hawtioPluginLoader ` since they are already globals anyway.